larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -317,10 +330,15 @@ TEST_P(BConv2DOpTest, SimpleTest) {
   const int stride_width = param.stride_width;
   const int dilation_height_factor = param.dilation_height_factor;
   const int dilation_width_factor = param.dilation_width_factor;
-  // TOOD: currently we are ignoring padding
   const Padding padding = param.padding;
+  const ActivationFunctionType activation = param.activation;
   const int num_threads = param.num_threads;
 
+  // Fused ReLu is not yet supported for zero-padding
+  if (padding == Padding_SAME && activation == ActivationFunctionType_RELU) {
+    return;
```

Comment:
this should return an error message.

Reply:
I agree, but then the unit tests would fail. I wanted to only skip this particular test combination but I did not know another way.

Reply:
in that case please create an issue that we don't forget bc with this test case I think we will see no failing tests for unsupported padding.

Reply:
I added this issue with a link to that line in the file: https://github.com/larq/compute-engine/issues/182